src/main/java/org/apache/commons/math4/random/SobolSequenceGenerator.java

```diff
@@ -62,10 +62,10 @@
     private static final double SCALE = FastMath.pow(2, BITS);
 
     /** The maximum supported space dimension. */
-    private static final int MAX_DIMENSION = 1000;
+    private static final int MAX_DIMENSION = 21201;
```

Comment:
Please update the "1000" to "21201" in the javadoc of SobolSequenceGenerator

Reply:
Done :)